src/service/ampdoc-impl.js

```diff
@@ -186,12 +172,9 @@ export class AmpDocService {
    *
    * An Error is thrown in development if no `AmpDoc` is found.
    * @param {!Node=} opt_node
-   * @param {{
-   *  closestAmpDoc: boolean
-   * }=} opt_options
    * @return {!AmpDoc}
    */
-  getAmpDoc(opt_node, opt_options) {
```

Comment:
Should also cleanup this usage:

https://github.com/ampproject/amphtml/blob/d93e3f00874dffde00558f89e07648ecf1fba445/extensions/amp-analytics/0.1/amp-analytics.js#L217-L219